gdal/doc/source/tutorials/geotransforms_tut.rst

```diff
@@ -0,0 +1,39 @@
+.. _geotransforms_tut:
+
+================================================================================
+Geotransform Tutorial
+================================================================================
+
+Introduction to Geotransforms:
+-------------------------------
+A geotransform is an affine transformation from the image coordinate space (row, column),
+also known as (pixel, line)
+to the georeferenced coordinate space (projected or geographic coordinates).
+
+A geotransform consists in a set of 6 coefficients:
+----------------------------------------------------
```

Comment:
This line with dashes should probably be removed as it appears as a title line and just replaced by a line with a line-feed. Please check that the rendering is OK

```suggestion

```